### ISSUE TITLE
vSRX Support?

### ISSUE THREAD

**Issue by kmarquis** [farkasmate/vagrant-junos_cli#3]
Im trying to use vagrant-junos_cli with vSRX, I set the .cmd file in the same directory as the Vagrantfile
```
Keerans-MBP:junos_sec kmarquis$ ls -la
total 64
drwxr-xr-x  17 kmarquis  staff   544 14 Mar 20:50 .
drwxr-xr-x   8 kmarquis  staff   256  3 Mar 09:28 ..
-rw-r--r--@  1 kmarquis  staff  6148 11 Mar 17:26 .DS_Store
drwxr-xr-x  12 kmarquis  staff   384 13 Mar 23:37 .git
-rw-r--r--   1 kmarquis  staff  2082 13 Mar 23:29 .gitignore
drwxr-xr-x   3 kmarquis  staff    96  2 Mar 21:35 .vagrant
-rw-r--r--   1 kmarquis  staff    60 13 Mar 23:26 README.md
-rw-r--r--   1 kmarquis  staff  4880 14 Mar 20:55 Vagrantfile
drwxr-xr-x   4 kmarquis  staff   128 14 Mar 20:42 config
drwxr-xr-x   7 kmarquis  staff   224  2 Mar 22:56 env
drwxr-xr-x   3 kmarquis  staff    96  3 Mar 09:29 group_vars
drwxr-xr-x   5 kmarquis  staff   160  3 Mar 09:16 host_vars
-rw-r--r--   1 kmarquis  staff     0  3 Mar 09:14 junos_sec.yml
-rw-r--r--   1 kmarquis  staff   534 13 Mar 23:23 requirement.txt
drwxr-xr-x   3 kmarquis  staff    96  3 Mar 09:45 roles
drwxr-xr-x   6 kmarquis  staff   192 11 Mar 17:26 scripts
-rw-r--r--@  1 kmarquis  staff  3332 14 Mar 20:36 srx1.cmd
```
however when try to provision the vSRX i get the error below
```
Keerans-MBP:junos_sec kmarquis$ vagrant provision srx1
==> srx1: Running provisioner: junos_cli...
    srx1: Running: /var/folders/15/56cwx1790g13kwvst2_wkjlc0000gn/T/vagrant-shell20180314-22409-kjdbik.cmd
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```
I additionally trying to move the .cmd file to another directory and i get the same error

My Vagrantfile looks like this
```# SRX 1
  config.vm.define "srx1" do |srx1|
    srx1.vm.network "private_network", virtualbox__intnet: "net1" # SRX1 - host1
    srx1.vm.network "private_network", virtualbox__intnet: "net2" # SRX1 - SRX2
    srx1.vm.network "private_network", virtualbox__intnet: "net5" # SRX1 - host3
    srx1.vm.box = "juniper/ffp-12.1X47-D15.4"
      srx1.vm.provider :virtualbox do |vb|
        vb.memory = 512
        vb.gui = false
      end
    srx1.vm.provision 'junos_cli', path: 'srx1.cmd'
  end 
```

**Comment by farkasmate** [comment]
Hi @kmarquis,

vSRX provisioning has been fixed in version 1.0.1, please update.

**Comment by kmarquis** [comment]
absolutely perfect 👍 cheers :)
